docs/BasicCommunication/OnPutFile/index.md

```diff
@@ -13,25 +13,31 @@ Purpose
 
 !!! MUST
 
-Use the appropriate uploaded file according to its workflow (IVSU, SystemRequest, RPC's). See diagrams listed below.
+1. Use the appropriate uploaded file according to its workflow (IVSU, SystemRequest, RPC's). See diagrams listed below.  
```

Comment:
the 1st diagram is a little misleading, the OnPutFile(---) from SDL to HMI happens after the Show(---) from mobile to SDL. There shall be no order of this. Suggest to move SHOW after OnPutFile

Reply:
@yang1070, thank you.
fixed  https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/97/commits/0f937ee25d5994c4838345b108997c496d2b2940